HotelBooking/Controllers/room_controller.py

```diff
@@ -17,42 +23,38 @@
 class RoomController(Controller):
     # TODO: Add customer validation logic when reserving/checking-in rooms
     room: Room
-    bill_controller: BillController
+    roomType: RoomType
+    reservation: Reservation
+    bill: Bill
 
     def __init__(self) -> None:
         super().__init__()
         self.room = Room()
-        self.bill_controller = BillController()
+        self.bill = Bill()
+        self.roomType = RoomType()
+        self.reservation = Reservation()
 
     def get_checked_in_rooms(self) -> List[Room]:
         return self.room.get_checked_in_rooms()
 
-    def get_reserved_rooms(self) -> List[Room]:
-        return self.room.get_reserved_rooms()
+    def get_reserved_rooms(self, userID=None) -> List[Room]:
```

Comment:
`user_id`

Reply:
actually can you change all references of `user_id` to `customer_id` so it's consistent with the database attribute `customer_id` so it does not confuse others?